autogpt/config/config.py

```diff
@@ -117,8 +120,7 @@ def __init__(self) -> None:
         # no user agent was found.
         self.user_agent = os.getenv(
             "USER_AGENT",
-            "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36"
-            " (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36",
+            "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36",
```

Comment:
This change may break things for people. If AutoGPT needs this to work on Windows, then I suggest we make it configurable, or set a different default based on sys.platform.